packages/ember-simple-auth-devise/README.md

```diff
@@ -137,12 +137,12 @@ In order to use the Devise authenticator (see the
 the application needs to have a login route:
 
 ```js
-App.Router.map(function() {
-  this.route('login');
+export default Router.map(function() {
+    this.route('login');
 });
 ```
 
-This route displays the login form with fields for `identification`,
+This route displays the login.hbs view with a form with fields for `identification`,
```

Comment:
login.hbs is the template, not the view - I think the original text should be sufficient actually.